google/cloud/storage/retry_policy.h

```diff
@@ -23,22 +23,28 @@ namespace google {
 namespace cloud {
 namespace storage {
 inline namespace STORAGE_CLIENT_NS {
+namespace internal {
```

Comment:
@coryan LGTM just wondering if this should be moved into storage/internal/ in the future?

Reply:
We can, but the style guide does not require namespaces to follow the directory hierarchy. I would gladly accept PRs :grin: